main.go

```diff
@@ -82,6 +82,672 @@ var (
 	masterMetrics = []snapshotMetric{
 		snapshotMetric{
 			desc: prometheus.NewDesc(
+				"mesos_master_cpus_percent",
+				"Percent of CPUs used as a fraction of 1.",
```

Comment:
In this sort of situation you'd try to keep them similar, and if there's big differences you can always put the original name in the help.